packages/react-core/src/components/Page/Page.tsx

```diff
@@ -81,6 +81,8 @@ export interface PageProps extends React.HTMLProps<HTMLDivElement> {
   isBreadcrumbGrouped?: boolean;
   /** Additional content of the group */
   additionalGroupedContent?: React.ReactNode;
+  /** HTML component used as main component of the page, defaults to 'main'. */
+  mainComponent?: 'main' | 'div';
```

Comment:
I might argue keeping it as `mainComponent` is good because
* There are a handful of props that are named `main[PropName]`, so it's more consistent
* If we did ever want to allow someone to change `div.pf-[v]-c-page`, it's probably best if that was done via `component` so we could run into namespacing issues

What's the reasoning for changing it to `component`?

Reply:
I was probably thinking of `mainComponent` as in the `main` tag, rather than "main" as in the base/primary/etc component. So "mainComponent" sounded a little off/confusing since the `main` element is one of the two types allowed. The naming is something we should align on, though, since we have a mix of "mainComponent" and just "component" for a prop.

I don't feel super strong about it, though, so if we'd want to revert this back I can push a quick change

Reply:
I would agree with Michael here.  I think we usually us `component` to change the outermost tag. Using `mainComponet` is more specific as to which tag/component  is actually being updated.

Reply:
+1, I like `mainComponent` here as well

Reply:
@thatblindgeye ah, gotcha! I believe the word here "main" refers to the BEM element name - `.pf-[v]-c-page__main`. If it were its own component, we would likely call it `<PageMain>` based off of the BEM name.